cardano-node-chairman/test/Spec/Chairman/Byron.hs

```diff
@@ -23,4 +23,4 @@ hprop_chairman = H.integration . H.runFinallies . H.workspace "chairman" $ \temp
   conf <- H.mkConf tempAbsPath' Nothing
   allNodes <- H.testnet conf
 
-  chairmanOver conf allNodes
+  chairmanOver 120 3 conf allNodes
```

Comment:
For Byron, if we expect a block every 2 seconds, we'd expect just under 60 blocks in 120 seconds, so our threshold ought to be 59 or 58 or so.

Reply:
Changing only `k` to `10` seems insufficient to getting Byron to produce more blocks.  It is currently still `3`.